### DISCUSSION TITLE
Parallel migrations in mysql

### DISCUSSION THREAD

**Discussion by navedSwyftx** [mikro-orm/mikro-orm#3332] [Q&A]
Is there a way to lock migration table in mysql, when running migrations. Currently, we are running multiple containers, which are trying to run migrations simultaneously. 

It is mentioned here https://github.com/mikro-orm/mikro-orm/issues/1623#issuecomment-810028629 but not sure whether it is actually implemented

**Comment by B4nan** [comment]
It is mentioned in that issue, but only about postgres. I am not aware how to do this with mysql. And nope, its not implemented even for postgres.

You should ensure the first container finishes the migration before you deploy others. You could use redis or some other fast storage to lock this explicitly.
